src/components/common/DataTag.js

```diff
@@ -0,0 +1,30 @@
+import CalendarToday from '@mui/icons-material/CalendarToday';
+import LocationOnOutlinedIcon from '@mui/icons-material/LocationOnOutlined';
+import { Box, Typography } from '@mui/material';
+import React from 'react';
+
+function DataTag({ data, location }) {
+  return (
+    <Box
+      mt={2}
+      mb={2}
+      sx={{
+        display: 'flex',
+        alignItems: 'center',
+        color: 'textLight.main',
+        opacity: 0.6,
+      }}
+    >
+      {location ? (
+        <LocationOnOutlinedIcon sx={{ mr: 1, fontSize: '1.25rem' }} />
+      ) : (
+        <CalendarToday sx={{ mr: 1, fontSize: '1.25rem' }} />
+      )}
+      <Typography varient="body1" sx={{ fontWeight: 700 }}>
+        {location || `Planter since`} {data}
+      </Typography>
+    </Box>
```

Comment:
```suggestion
    <Stack
      gap={1}
      direction='row'
      sx={{
        alignItems: 'center',
        color: 'textLight.main',
        opacity: 0.6,
      }}
    >
      {location ? (
        <LocationOnOutlinedIcon sx={{ fontSize: '1.25rem' }} />
      ) : (
        <CalendarToday sx={{ fontSize: '1.25rem' }} />
      )}
      <Typography variant="body1" sx={{ fontWeight: 700 }}>
        {location || `Planter since`} {data}
      </Typography>
    </Box>
```